pkgs/applications/networking/mumble/default.nix

```diff
@@ -25,7 +25,7 @@ let
     patches = (source.patches or [])
       ++ [ ./fix-rnnoise-argument.patch ];
 
-    nativeBuildInputs = [ pkg-config python qt5.qmake ]
+    nativeBuildInputs = [ pkg-config python3 qt5.qmake ]
```

Comment:
Shouldn't this use libsForQt5 instead of passing qmake in?

Reply:
In all-packages.nix mumble and murmur use callPackages but there is no libsForQt5.callPackages.

Reply:
I mean ``libsForQt5.callPackage``.

Reply:
Is it safe to replace callPackages with libsForQt5.callPackage?

Reply:
I am not sure. It could double wrap maybe.

Reply:
This sounds like something that should be handled in a separate pull request.